utils/check-style/check-mypy

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# The mypy supports pyproject.toml, but unfortunately it doesn't support it recursively
```

Comment:
Just curious: I couldn't find any pyproject.toml files in our repository. Is "pyproj.toml" support even relevant to mypy? Such files don't contain Python code, right?

Reply:
Uh.